qtfred/src/ui/widgets/sexp_tree.cpp

```diff
@@ -5177,6 +5182,43 @@ sexp_list_item* sexp_tree::get_listing_opf_asteroid_debris()
 	return head.next;
 }
 
+sexp_list_item* sexp_tree::get_listing_opf_lua_enum(int parent_node, int arg_index)
+{
+	int this_index = get_dynamic_parameter_index(tree_nodes[parent_node].text, arg_index);
+
+	// first child node
+	int child = tree_nodes[parent_node].child;
+	if (child < 0)
+		return nullptr;
+
```

Comment:
Just a note on code like this. If we were in a gameplay function, it would be best to move the `get_dynamic_parameter_index` function call below the early return check.  Here it basically doesn't matter, unless FRED or qtFRED end up being wildly different than initially designed in the future.

If you were to move it, it would make the could a bit clearer (assignment closer to variable use is generally better for readability), but I am far from requiring that change.